### PR TITLE
Bael 4781: Introduction to ZeroCode

### PR DESCRIPTION
**Pull request by sparrowV**

### CODE REVIEW

File: testing-modules/pom.xml [hunk 1]
```diff
@@ -1,6 +1,5 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (KevinGilmore) [inline testing-modules/pom.xml:2]: Please refrain from making changes that merely reformat or rearrange items like this,

File: testing-modules/zerocode/src/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+## Introduction to Zerocode
```
Comment (KevinGilmore) [inline testing-modules/zerocode/src/README.md:1]: You can remove this file for now -- I will create it after the article is published.

**Review by KevinGilmore — CHANGES_REQUESTED** [review]
Please see notes about the formatting changes in pom.xml, remove the README.md file, and question: do you need the empty application.properties file? If not, let's remove it.

File: testing-modules/pom.xml (hunk 1 above)
Comment (sparrowV) [reply testing-modules/pom.xml:2]: sure, I don't know how why the header was changed.  Should I leave like this for now, or revert the commit and commit again?

Comment (KevinGilmore) [reply testing-modules/pom.xml:2]: You could just edit the file to restore the original format to those lines, then commit and push again.

**Review by KevinGilmore — APPROVED** [review]

**Pull request merged**